src/darwin/Framework/CHIP/MTROperationalCredentialsDelegate.h

```diff
@@ -55,6 +57,18 @@ class MTROperationalCredentialsDelegate : public chip::Controller::OperationalCr
     void SetDeviceID(chip::NodeId deviceId) { mDeviceBeingPaired = deviceId; }
     void ResetDeviceID() { mDeviceBeingPaired = chip::kUndefinedNodeId; }
 
+    void SetDeviceCommissioner(chip::Controller::DeviceCommissioner * cppCommissioner) { mCppCommissioner = cppCommissioner; }
```

Comment:
Please document what keeps this from becoming a dangling pointer, or at least ensures it's not touched if it does become dangling.

Reply:
It will not get touched in case the actual CppCommissioner is destroyed because this pointer is only dereferenced when the commissioner is commissioning a device. For that the MTRDeviceController and cppCommissioner inside it have to be valid objects.

Reply:
Added some documentation to state this in the code.